assets/base.css

```diff
@@ -177,13 +177,15 @@ html.no-js .no-js-hidden {
   padding: 0 1.5rem;
 }
 
-.page-width-desktop {
+.page-width-desktop,
+.page-width-tablet {
   padding: 0;
   margin: 0 auto;
 }
 
 @media screen and (min-width: 750px) {
-  .page-width {
+  .page-width,
+  .page-width-tablet {
```

Comment:
and here it could be `.page-width-tablet.page-width-desktop`

Reply:
I don't like that this makes the tablet class unusable without the desktop class, but even if we did `.page-width-tablet.page-width-tablet` instead, adding specificity here would actually override the desktop breakpoint styles for `.page-width-desktop`. Currently that's fine because the padding value is the same for both, but that could theoretically change. The least destructive way I saw to safely preserve the current inheritance is to add an separate rule for `.page-width-tablet` below the `.page-width-desktop` rule